src/Microsoft.Kusto.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -270,6 +269,24 @@ public virtual async Task<ConnectionCompleteParams> Connect(ConnectParams connec
             return completeParams;
         }
 
+        internal void RefreshAzureToken(string ownerUri)
+        {
+            ConnectionInfo existingConnection = OwnerToConnectionMap[ownerUri];
+            
+            var requestMessage = new RequestSecurityTokenParams
+            {
+                Authority = existingConnection.ConnectionDetails.GetOptionValue("azureTenantId", string.Empty),
+                Provider = "Azure",
+                Resource = "SQL"
```

Comment:
Whys is the resource "SQL" here?

Reply:
I reached out to Amir to verify this. The ID of the resource Url we're hitting is called 'sql'. It's defined within providerSettings.ts within ADS. It has to be 'sql' otherwise it'll throw an error. Here's how it's defined in ADS.
`sqlResource: {
				id: 'sql',
				endpoint: 'https://database.windows.net/',
				azureResourceId: AzureResource.Sql
			},`

Reply:
Please add a comment on this.

Reply:
Can we get these strings from the DataSource, or atleast define them in the DataSource directory. In the future if we need to update this, the reference regarding authentication params being used will be easy to find.